visidata/loaders/mysql.py

```diff
@@ -0,0 +1,114 @@
+from visidata import *
+
+__all__ = ['openurl_mysql', 'MyTable', 'MyTablesSheet']
+
+def codeToType(type_code, colname):
+    import MySQLdb as mysql
+
+    if type_code == mysql.NUMBER:
+        return int
+
+    if type_code == mysql.STRING:
+        return str
+
+    return anytype
+
+
+def openurl_mysql(url, filetype=None):
+    import MySQLdb as mysql
+
+    url = urlparse(url.given)
+    dbname = url.path[1:]
+    conn = mysql.connect(
+                user=url.username,
+                database=dbname,
+                host=url.hostname,
+                port=url.port or 3306,
+                password=url.password,
+                use_unicode=True,
+                charset='utf8')
+
+    return MyTablesSheet(dbname+"_tables", sql=SQL(conn), schema=dbname)
+
+
+class SQL:
+    def __init__(self, conn):
+        self.conn = conn
+
+    def cur(self, qstr):
+        # FIXME: The Postgres loader is using a named cursor but afaik MySQL does not know named cursors… problem?
```

Comment:
Just to be sure, because I don’t know enough about async Python: could a named cursor be necessary for the methods decorated with `@asyncthread`? If it was introduced because of those in the Postgres loader, it might be necessary to rewrite the MySQL loader synchronously…?

Reply:
I guess it might be true, but that seems like a per-loader kind of thing. And we don't have multiple threads accessing the db at the same time; they may be different threads, but they are completely isolated in time.  So as long as it works, we don't need it yet.

Reply:
(So let's just remove the comment).

Reply:
Fixed in 5151db12db9ba71de0807219bbf14930192df408.